package.json

```diff
@@ -42,10 +42,10 @@
     "sqs-consumer": "^3.0.0",
     "stream-meter": "^1.0.3",
     "superagent-promise": "^1.0.3",
-    "taskcluster-lib-api": "^0.12.0",
+    "taskcluster-lib-api": "^1.0.0",
     "taskcluster-lib-app": "^0.8.10",
     "taskcluster-lib-loader": "^0.1.1",
-    "taskcluster-lib-stats": "^0.8.8",
+    "taskcluster-lib-monitor": "^2.0.0",
```

Comment:
You'll need [version 2.1.0](https://github.com/taskcluster/taskcluster-lib-monitor/releases/tag/v2.1.0) for `process` to do anything in the constructor. I suppose this will just work, but may as well be precise while we're at it.